src/rpc/RPCHelpers.cpp

```diff
@@ -1539,48 +1539,42 @@ traverseTransactions(
     }
 
     auto minIndex = context.range.minSequence;
+    auto maxIndex = context.range.maxSequence;
+    std::optional<int64_t> min;
+    std::optional<int64_t> max;
+
     if (request.contains(JS(ledger_index_min)))
     {
-        auto& min = request.at(JS(ledger_index_min));
+        if (request.at(JS(ledger_index_min)).is_int64())
+            min = request.at(JS(ledger_index_min)).as_int64();
 
-        if (!min.is_int64())
+        else
             return Status{
                 RippledError::rpcINVALID_PARAMS, "ledgerSeqMinNotNumber"};
 
-        if (min.as_int64() != -1)
```

Comment:
Don't we still need to support this -1 case?

Reply:
@godexsoft you're right as we pivoted away from unit64 to int64, so this check becomes necessary again